index.js

```diff
@@ -0,0 +1,53 @@
+let bookCollection = localStorage.getItem('bookCollection')
+  ? JSON.parse(localStorage.getItem('bookCollection'))
+  : [];
+const title = document.querySelector('#title-input');
+const author = document.querySelector('#author-input');
+const bookContainer = document.querySelector('.books-container');
+
+const removeBook = function () {
+  const title = this.id.split('-')[0];
+  const author = this.id.split('-')[1];
+  bookCollection = bookCollection.filter(
+    (book) => book.title !== title && book.author !== author,
+  );
```

Comment:
- [ ] I would suggest when adding a book you also pass a given ID to it, because now if we have two books with the same author and title and I try to remove one of them, both of them get removed, that should not be the case. Please update it